activesupport/lib/active_support/i18n_railtie.rb

```diff
@@ -4,6 +4,22 @@
 
 module I18n
   class Railtie < Rails::Railtie
+    module Helpers
+      def self.extensions_for(glob)
+        $1.split(',') if glob =~ /\{([\S]+)\}/
```

Comment:
It might even be a good idea to research how picking up new/removed files in the `app` directory is done :wink: